crates/engine/primitives/Cargo.toml

```diff
@@ -24,6 +24,7 @@ reth-errors.workspace = true
 alloy-primitives.workspace = true
 alloy-consensus.workspace = true
 alloy-rpc-types-engine.workspace = true
+alloy-eips = { workspace = true, features = ["kzg"] }
```

Comment:
This shouldn't need the kzg feature if it's just validating requests

Reply:
```suggestion
alloy-eips.workspace = true
```

Reply:
wonder if there is any tool to check needlessly enabled features in ci @Rjected @DaniPopes